README.md

```diff
@@ -93,4 +117,12 @@ Now we need set up the repository. Create new repository or use an existing one.
 3. Copy your smee.io url into Payload URL field
 4. Select content-type ```application/json```
 5. Input any "secret" word or phrase into the Secret field.
-6. In the block 'Which events would you like to trigger this webhook?' select 'Let me select individual events' and check the events as listed in the Usage section.
\ No newline at end of file
+6. In the block 'Which events would you like to trigger this webhook?' select 'Let me select individual events' and check the events as listed in the Usage section.
+
+### Debugging behaviour in container
+
+To debug app behaviour in container environment you can use docker-compose-debug.yaml (it will pick latest version of labeller app from image repository):
+```
+docker compose -f docker-compose-debug.yml up -d
+```
+Or build Dockerfile yourself to try out your local code.
```

Comment:
Feels kinda sus, unsure. Shouldn't local docker compose files *not* use the images from the registry? You would want to test your own changes, no?